feature/experimental/p4rt/ate_tests/google_discovery_protocol_packetin_test/gdp_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package google_discovery_protocol_packetin_test
```

Comment:
I see that there are three files here gdp_test, google_discovery_protocol_packetin_test and packetin_test. Can we combine gdp_test and google_discovery_protocol_packetin_test into one.  gdp_test has hardly four functions, it would be better if we could merge them into google_discovery_protocol_packetin_test.

Reply:
Sure. I will merge gdp_test into google_discovery_protocol_packetin_test